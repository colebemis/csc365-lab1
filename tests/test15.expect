Enter query: 0: 0
1: 2
2: 13
3: 9
4: 15
5: 0
6: 22
Enter query: 