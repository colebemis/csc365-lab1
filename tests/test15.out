Enter query: 0: 0
1: 2
2: 13
3: 9
4: 15
5: 0
6: 21
Enter query: 